internal/pkg/manifest/testdata/backend-svc-customhealthcheck.yml

```diff
@@ -37,4 +37,6 @@ exec: true     # Enable running commands in your container.
 # You can override any of the values defined above by environment.
 #environments:
 #  test:
-#    count: 2               # Number of tasks to run for the "test" environment.
\ No newline at end of file
+#    count: 2               # Number of tasks to run for the "test" environment.
+#    deployment:            # deployment strategy for the "test" environment.
+#       rolling: 'recreate' # key is rolling deployment and value is deployment strategy name i.e. either default or recreate.
```

Comment:
I think the explanation may be confusing to user. We want to explain what "recreate" is, something like "Existing tasks are stopped before new ones are started"